index.html

```diff
@@ -35,7 +35,7 @@ <h2>
 
 <p>rdiff-backup backs up one directory to another, possibly over a network. The target directory ends up a copy of the source directory, but extra reverse diffs are stored in a special subdirectory of that target directory, so you can still recover files lost some time ago. The idea is to combine the best features of a mirror and an incremental backup. rdiff-backup also preserves subdirectories, hard links, dev files, permissions, uid/gid ownership (if it is running as root), modification times, acls, eas, resource forks, etc. Finally, rdiff-backup can operate in a bandwidth efficient manner over a pipe, like rsync. Thus you can use rdiff-backup and ssh to securely back a hard drive up to a remote location, and only the differences will be transmitted. </p>
 
-<p>Sol1 has officially taken over stewardship of rdiff-backup from February 2016. Sol1 has long been a contributor and user of rdiff-backup, and will maintain the open source nature of rdiff-backup, while bringing it into the modern era.</p>
+<p><a href="https://sol1.com.au/">Sol1</a> has officially taken over stewardship of rdiff-backup from February 2016. <a href="https://sol1.com.au/">Sol1</a> has long been a contributor and user of rdiff-backup, and will maintain the open source nature of rdiff-backup, while bringing it into the modern era.</p>
```

Comment:
You mind if I also add a link to Seravo.com somewhere here?

Reply:
You could always add few lines about the contribution of seravo as being part of the new "team". I should probably do the same for my business. Me I don't mind. Having businesses behind an open source project is rewarding.

Reply:
Fine with me as well.

Reply:
Maybe we can have a contributors past and present section? PS, you guys are doing an amazing job! great work!